tools/__init__.py

```diff
@@ -337,9 +337,9 @@ def verify_install(self):
     def _attempt_install(self):
         try:
             # check for presence of conda command
-            util.misc.run_and_print(["conda", "-V"], silent=True, check=True, env=self.conda_env)
+            util.misc.run_and_print(["conda", "-V"], loglevel=logging.WARNING, check=True, env=self.conda_env)
```

Comment:
`run_and_print` logs at that level regardless of success or failure, so `logging.WARNING` might be a too verbose. `logging.INFO` could be good to differentiate between that and `DEBUG` level logging.